src/user.py

```diff
@@ -122,6 +122,27 @@ def display_default(values, _):
 
     return {"users": users}
 
+def list_shells():
+    import ctypes
+    import ctypes.util
+
+    """List the shells from /etc/shells."""
+    libc = ctypes.CDLL(ctypes.util.find_library("c"))
+    getusershell = libc.getusershell
+    getusershell.restype = ctypes.c_char_p
+    libc.setusershell()
+    while True:
+        shell = getusershell()
+        if not shell:
+            break
+        yield shell.decode()
+    libc.endusershell()
```

Comment:
Uuuuh I guess that works but that looks pretty complex ? :thinking: 

Why not just read `/etc/shells` directly ? (I guess we just need a trick to ignore comments / empty lines ?)

Reply:
done :)